tensorboard/plugins/graph/tf_graph_node_search/BUILD

```diff
@@ -16,3 +16,15 @@ tf_web_library(
         "@org_polymer_paper_styles",
     ],
 )
+
+tensorboard_webcomponent_library(
```

Comment:
You'll need `load("//tensorboard/defs:defs.bzl", "tensorboard_webcomponent_library")` for this, right?  Looks like we shouldn't have removed that in #1161 after all.

Reply:
Done